Files qmail-1.03.orig/auth_imap and qmail-1.03/auth_imap differ
Files qmail-1.03.orig/auth_pop and qmail-1.03/auth_pop differ
Files qmail-1.03.orig/auth_smtp and qmail-1.03/auth_smtp differ
diff -NurpP qmail-1.03.orig/control.c qmail-1.03/control.c
--- qmail-1.03.orig/control.c	2008-04-06 19:06:15.683511000 +0000
+++ qmail-1.03/control.c	2008-04-06 19:15:47.923622781 +0000
@@ -1,3 +1,6 @@
+#define _GNU_SOURCE 1
+#include <string.h>
+
 #include <unistd.h>
 #include "readwrite.h"
 #include "open.h"
@@ -10,10 +13,6 @@
 #include "scan.h"
 #include "limit.h"
 
-#if defined(USE_CONTROLDB) || defined(SECUREBIND_SASL) || defined(SECUREBIND_SSL) || defined(SECUREBIND_TLS) || defined(SECUREBIND_ALL)
-#include <string.h>
-#endif
-
 #ifdef USE_CONTROLDB
 #include <syslog.h>
 
@@ -48,7 +47,6 @@ int control_ldap_search(const char *fn, 
 
 #if defined(SECUREBIND_SASL) || defined(SECUREBIND_SSL) || defined(SECUREBIND_TLS) || defined(SECUREBIND_ALL)
 #include <malloc.h>
-#include <string.h>
 #include <malloc.h>
 
 #include "subfd.h"
Files qmail-1.03.orig/control.o and qmail-1.03/control.o differ
Files qmail-1.03.orig/pbsadd and qmail-1.03/pbsadd differ
Files qmail-1.03.orig/pbscheck and qmail-1.03/pbscheck differ
Files qmail-1.03.orig/pbsdbd and qmail-1.03/pbsdbd differ
Files qmail-1.03.orig/qmail-forward and qmail-1.03/qmail-forward differ
Files qmail-1.03.orig/qmail-group and qmail-1.03/qmail-group differ
Files qmail-1.03.orig/qmail-inject and qmail-1.03/qmail-inject differ
Files qmail-1.03.orig/qmail-ldaplookup and qmail-1.03/qmail-ldaplookup differ
Files qmail-1.03.orig/qmail-lspawn and qmail-1.03/qmail-lspawn differ
Files qmail-1.03.orig/qmail-newmrh and qmail-1.03/qmail-newmrh differ
Files qmail-1.03.orig/qmail-pw2u and qmail-1.03/qmail-pw2u differ
Files qmail-1.03.orig/qmail-qmqpc and qmail-1.03/qmail-qmqpc differ
Files qmail-1.03.orig/qmail-qmtpd and qmail-1.03/qmail-qmtpd differ
Files qmail-1.03.orig/qmail-queue and qmail-1.03/qmail-queue differ
Files qmail-1.03.orig/qmail-quotawarn and qmail-1.03/qmail-quotawarn differ
Files qmail-1.03.orig/qmail-remote and qmail-1.03/qmail-remote differ
Files qmail-1.03.orig/qmail-reply and qmail-1.03/qmail-reply differ
Files qmail-1.03.orig/qmail-secretary and qmail-1.03/qmail-secretary differ
Files qmail-1.03.orig/qmail-send and qmail-1.03/qmail-send differ
Files qmail-1.03.orig/qmail-showctl and qmail-1.03/qmail-showctl differ
Files qmail-1.03.orig/qmail-smtpd and qmail-1.03/qmail-smtpd differ
Files qmail-1.03.orig/qmail-todo and qmail-1.03/qmail-todo differ
Files qmail-1.03.orig/qmail-verify and qmail-1.03/qmail-verify differ
